src/content/app/help/Help.tsx

```diff
@@ -92,14 +120,82 @@ const AppBar = () => {
   );
 };
 
+const NavButtons = () => {
```

Comment:
As we discussed at code review, this looks like it contains what can be a shared component with the following interface:

```ts
type HistoryButtonsProps = {
  onHistoryBack: () => void;
  onHistoryForward: () => void;
  hasPrevious: boolean
  hasNext: boolean
}

const HistoryButtons = () => { ... }
```

The Help component, then, will contain a component that will wrap the `HistoryButtons` component; for example `HelpHistory` or something. This component will read data from the redux state and pass it to the `HistoryButtons` component.

Reply:
I have pushed the changes for this ✅